src/garage/sampler/on_policy_vectorized_sampler.py

```diff
@@ -34,14 +43,33 @@ def start_worker(self):
 
         self.vec_env = VecEnvExecutor(
             envs=envs, max_path_length=self.algo.max_path_length)
-        self.env_spec = self.env.spec
 
     def shutdown_worker(self):
         """Shutdown workers."""
         self.vec_env.close()
 
+    # pylint: disable=too-many-statements, import-outside-toplevel
```

Comment:
please move `import time` to the top of the file. there's no need for it to be a late import.